seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/common/AbstractSeaTunnelTransform.java

```diff
@@ -105,4 +141,39 @@ public R transform(SeaTunnelRow row) {
     protected abstract TableSchema transformTableSchema();
 
     protected abstract TableIdentifier transformTableIdentifier();
+
+    @Override
+    public void setMetricsContext(MetricsContext metricsContext) {
+        this.metricsContext = metricsContext;
+    }
+
+    @Override
+    public MetricsContext getMetricsContext() {
+        return metricsContext;
+    }
+
+    protected void hazelcastMetric(long size) {
+        if (metricsContext != null) {
+            metricsContext.counter(getTransformMetricName(this)).inc(size);
+        }
+    }
+
+    protected void hazelcastMetric() {
+        if (metricsContext != null) {
+            metricsContext.counter(getTransformMetricName(this)).inc();
+        }
```

Comment:
is more concise and easier to read
```suggestion
      hazelcastMetric(1)
```